tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx",
+    "jsxImportSource": "dom-renderer",
+    "experimentalDecorators": true,
+    "useDefineForClassFields": false
```

```suggestion
    "useDefineForClassFields": true
```
WebCell v3 使用 stage-3 版的最新装饰器语法，请仔细看 ReadMe。